pkg/option/config.go

```diff
@@ -2387,6 +2387,13 @@ func (c *DaemonConfig) TunnelingEnabled() bool {
 	return c.Tunnel != TunnelDisabled
 }
 
+// TunnelExists returns true if pod-to-pod traffic may go through a tunnel,
```

Comment:
```suggestion
// TunnelExists returns true if inter-node traffic may go through a tunnel,
```

("pod-to-pod" doesn't apply to the EgressGW)